client/app/src/components/account/exchange-tab.controller.js

```diff
@@ -16,9 +16,58 @@
       controller: ExchangeTabController
     })
 
-  function ExchangeTabController ($scope) {
+  function ExchangeTabController ($scope, $sce) {
     this.$onInit = () => {
       this.ul = this.accountCtrl
+
+      const merchantId = 'bab9de3731aa'
+      const refId = merchantId
+
+      let from
+      let to = 'ARK'
+      let address = ''
+      let amount
+
+      if (this.ul.network.token === 'DARK') {
+        from = 'btc'
+        amount = 0.1
+      } else {
+        to = this.ul.network.token
+
+        // Use the current address to receive the ARKs
+        if (to === 'ARK') {
```

Comment:
This can be removed

Reply:
This was a way to make the wallet slightly more compatible to other ARK-based chains